.eslintrc

```diff
@@ -25,6 +25,6 @@
         "es6": true,
         "node": true,
         "browser": true,
-        "mocha": true
+        "jasmine": true
```

Comment:
You probably want to turn this on only in the tests. You can do that by putting `{"env": {"jasmine": true}}` in `tests/.eslintrc`. It will automatically inherit from the main one.